Readme.md

```diff
@@ -25,6 +25,60 @@ To start a local Ethereum node with the contracts deployed, execute:
 This will create a `deployment-localhost.json` file containing the addresses of
 the deployed contracts.
 
+To deploy the marketplace you can define `HARDHAT_NETWORK` and run the following command:
+
+```bash
+npm run deploy
```

Comment:
```suggestion
npm run deploy -- --network localhost
```

This is btw. a way how you can pass other parameters to the underlying `deploy` script, so you can remove the "Or you can use npx directly" cases. That is btw. why I have asked you to remove originally the `--network localhost` from the `deploy` script because you can then specify it yourself.